### PR TITLE
Fixes hisspering (whisper bug)

### PR DESCRIPTION
**Pull request by SvartaSvansen**
Have you ever had this happen to you? A lizard walks up to you and tells you a secret, but does so with some unnecessssssssssssssssssarily sssssssssibiliant whissssssssspering in your ear? Annoying? Yes! Rude? You bet!

Well, fear not! Soon this hisssssssssssssssssspering phenomenon will be a thing of the past! We'll send those unruly lizards to mandatory classes in reptilian whispering etiquette and teach them how not to blow their spittle into others' ears and cause them headaches! You're a Nanotrasen employee for heaven's sake, not a leaky tire!

Note: Not all lizards hissper. Only those that whisper using :w. Hisspering is where one "s" turns into nine instead of three and is the result of say() and whisper() both processing the message with speech effects. Slurring, stuttering, and derpspeech are processed twice as well.

Note: Sorry to those lizards who find hisspering sexy. It's not worth the headaches of the whole rest of the crew.

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -90,7 +90,9 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
 		src << "<span class='warning'>You find yourself unable to speak!</span>"
 		return
 
-	message = treat_message(message)
+	if(message_mode != MODE_WHISPER) { //whisper() calls treat_message(); double process results in "hisspering"
```
Comment (RemieRichards) [inline code/modules/mob/living/say.dm]: ```{``` are not required in this language, please remove for consistency.

**Comment by Cheridan** [comment]
For future reference, we generally don't changelog bugfixes unless they were particularly game-destroying and it's necessary or important to notify players of its crushing.

**Comment by SvartaSvansen** [comment]
Ack. My bad. I'll take that off. Didn't know!

**Comment by Antimattercarp** [comment]
I learned a new word today!

**Pull request merged**